src/game.cpp

```diff
@@ -1556,6 +1556,35 @@ unsigned int game::get_seed() const
     return seed;
 }
 
+void game::set_highway_global_offset( const int &x, const int &y )
```

Comment:
I guess clang-tidy is going to complain and suggest to use `point` to store the offset instead.

Reply:
Does point make sense in an overmap context? (As in the x and y values are whole overmaps)

Reply:
Sounds like a use case of `point_rel_om`

See https://github.com/CleverRaven/Cataclysm-DDA/blob/master/doc/POINTS_COORDINATES.md#point-types

Reply:
Oh right I forgot there are point versions of all of those